### DISCUSSION TITLE
Video Ordner in einem Video Ordner übernimmt nicht die Einstellungen, wo ich eingestellt habe.

### DISCUSSION THREAD

**Discussion by Conan179** [LesFerch/WinSetView#134] [General]
Guten Tag.
Es geht um folgendes.
Ich hab für Video Ordner eine andere Einstellungen, wie für Allgemeine.
Mir ist aufgefallen, das meine Einstallung im Ordner übernommen werden, aber bei einem Subfolder nicht.
Windows sagt, der Ordner Typ sei Video.
Kann man heraus finden, warum meine Einstellungen nicht angewendet werden?
![ordner_ansichten_111124](https://github.com/user-attachments/assets/ff326d0c-1470-414d-94c8-6157a3497e3f)

**Comment by LesFerch** [comment]
WinSetView deletes all of the existing folder views. Therefore, all folders of the same type must open with the new default view. The only way I can see that not happening is if another program is running that restores old view data.

Do you have any program installed that manages Explorer windows, such as [WindowManager](https://www.desksoft.com/WindowManager.htm)?

**Reply by Conan179** [reply]
Nein.

**Comment by LesFerch** [comment]
I can't replicate this issue, but let me set up a test with everything as close to your configuration as possible.

It looks like you're running Windows 11 with Open Shell installed. Correct?

Did you do a portable install of WinSetView or use the setup exe?

Are you running as a Standard user or Administrator user?

Any other software installed that might be relevant?

Is RegEdit disabled on your computer?

**Reply by Conan179** [reply]
Ja, Ich nutzte Windows 11 24h2, Ok die openshell hatte ich vergessen, das ich die innstalliert habe, benutzte die nur für zusätzliche knöpfe, die man auf den screenshot sehen kann.
Die setup.exe, Standartuser, Ich benutzte den PC allein, nein ich RegEdit ist nicht deaktiviert.

**Comment by LesFerch** [comment]
I have already tested with a standard user, setup exe, and Windows 11 24H2. I'll go ahead and add Open Shell, but I cannot see how that would make a difference.

Is this page set to these defaults?
![image](https://github.com/user-attachments/assets/0dfcd607-4196-43c2-8efd-72a5f41050bb)

**Reply by Conan179** [reply]
Die Option "Erzwingen Sie keine Standardgruppierung für Bibliotheken" war an, hab sie aus gemacht, 
Ändert leider nichts.

**Comment by LesFerch** [comment]
Does it make a difference if you choose a different view for Videos, such as Details or Tiles? Does that view apply to all other folders of type Videos?

**Reply by Conan179** [reply]
Nein, es macht keinen unterschied.
Hab nach geschaut,jetzt sind alle "Video" Ordners, wie bei der einstellung "Gobal"...

**Reply by LesFerch** [reply]
It seems to be behaving like "Make all folders Generic" is selected except that would make the folder type "General items" and your folders are type "Videos". Let's find out if this is a profile-specific issue or something system-wide. Can you [create another local account](https://www.elevenforum.com/t/add-local-user-account-in-windows-11.590/) (make it an administrator) and login with that account and see if it then works correctly?

Also, let's confirm that your existing views are being cleared. After running WinSetView, open RegEdit to this key (paste into RegEdit's navigation bar):

`HKCU\Software\Classes\Local Settings\Software\Microsoft\Windows\Shell\Bags`

There should only be a small number of `Bags` entries (typically only 2 if no folders have been opened since running WinSetView):

![image](https://github.com/user-attachments/assets/9261d815-26a0-46f6-a531-d815fa97e531)

**Reply by Conan179** [reply]
Ich kam noch nicht dazu das zu testen.

**Reply by Conan179** [reply]
Ok, das ist doof, mit dem Test konto funktioniert es. Was kann ich jetzt machen?

**Reply by LesFerch** [reply]
That's a great relief for me. There's no good explanation for it not to work other than something being corrupted.

Usually any folder views issue caused by registry corruption are corrected by WinSetView because it resets the BagMRU/Bags key pair. That's why I asked about the Bags key. Is it being cleared for your account as I showed in the screenshot above?

You said that your account is a "Standard" user. Try changing your account to an "Administrator" user and test again. If that fixes it, then change it back to a Standard user and see if the problem comes back.

**Reply by Conan179** [reply]
Oh du meintest mit "Administrator user" nicht den seperaten Admin Acc wo man standartmässig nicht sehen kann?
Ich meinte mit "Standard" user den wo man normal nutzt und der hat Administrator rechte.

**Reply by LesFerch** [reply]
No, I didn't mean that. I understand the confusion. A typical Windows user on a personally owned computer is usually account type "Administrator". Users on managed work computers are typically logged in as more restricted "Standard" users. WinSetView supports both types. Since your account is already type "Administrator", then that's not a concern.

We'll need to focus on corruption and/or unusual registry settings that might cause the issue.

**Reply by Conan179** [reply]
Ok, das hatte ich falsch verstanden gehabt 😅

Wen du mir sagst, was ich dir schicken soll, schicke ich sie dir gern.

**Reply by LesFerch** [reply]
If you can send me an export of HKCU, I might be able to spot the problem. In RegEdit, right-click HKEY_CURRENT_USER, select Export and save it to a file. Zip that REG file and then send it via [wormhole](https://wormhole.app/). [Email](https://lesferch.github.io/WinSetView/Manual.html) me the link wormhole provides.

**Reply by LesFerch** [reply]
You may as well send me your **win10.ini** file, although I don't expect any issue there since it worked for the test user. I think you installed to `C:\Program Files (x86)`, so the file should be `%appdata%\WinSetView\AppData\Win10.ini`

**Reply by Conan179** [reply]
Ok, die Mail ist raus

**Reply by LesFerch** [reply]
I created a Tobias local user account, imported your HKCU file, and tested using your Win10.ini file. Everything worked correctly. The only folders that opened in the General items view were folders that were type General items.

There may be some software running for your account that I don't have, and your test account doesn't have, that's causing the trouble. Try starting up in Safe Mode and see if that makes a difference:

Hold Shift while selecting Restart
Troubleshoot
Advanced Options
Startup Settings
Restart
4) Enable Safe Mode

Once in Safe Mode, run WinSetView again and check folders on all drives.

P.S. When I looked at your HKCU reg file, I was surprised by the huge and unusual entries created by Start 8. Although those keys didn't cause any trouble in my tests, they do concern me. Are you still running Start 8 or are those keys just a leftover? If they're leftovers, I would delete them.

Another unusual key in your profile is:
`HKEY_CURRENT_USER\Software\29D873AE71F6A5673BE1`
That key contains one entry named `1A717C40FF7F60C18953B46A69A8FC47CCE7DAD6116CD3715DEB2ABF0D80722D` that goes on for over 6000 lines of hex data.

**Reply by Conan179** [reply]
Start 8? Nein nutzte ich nicht, aber ich gehe davon aus, das die Nachfolger Software https://www.stardock.com/products/start11/ schlecht programmiert worden ist und die einträge erstellt, weil es von dem Hersteller https://www.stardock.com/products/start8/ gibt.
Den HKEY_CURRENT_USER\Software\29D873AE71F6A5673BE1 habe ich gelöscht und beobachte, ob sich was ändert.

**Reply by Conan179** [reply]
Ok, das ist mir jetzt echt peinlich, ich bi mir so gut wie sicher, das der Übeltäter https://github.com/Open-Shell/Open-Shell-Menu ist.
Nicht die Software an sich, sondern die Settingsdatei wo ich bei einer neu innstallation importiere...
Beim abgesicherten Modus, war der Fehler nicht da, also habe ich probiert, die settings zurück zu setzten und der fehler trat nicht auf.
Ich muss aber dazu sagen, die Einstellungen nutzte ich seid mehr als 5 Jahren 
![settings_141124aa](https://github.com/user-attachments/assets/2537d46d-894d-4f91-be41-1cbadb296ead)
Ich werde jetzt die Standart einstellungen nutzten und nur die Icons, wo ich brauche in die Leiste bringen und nichts anderes ändern und beobachten.

Ich endschuldige mich für die unannehmlichkeiten und Danke ihnen für die Hilfe.

**Reply by LesFerch** [reply]
I'm curious about what's in the XML file that would cause that problem. Can you send me a copy of it?

**Reply by Conan179** [reply]
Brauch ich nicht, der Fehler ist nicht in der Datei sondern es ist die Option "Show sort headers in all views".
Wen die an ist, ist der Fehler da, wen sie aus ist, ist der Fehler nicht da.
Ich hab einen Bug Report dort eröffnet, mal schauen ob das gefixt wird...
https://github.com/Open-Shell/Open-Shell-Menu/issues/2023

**Reply by LesFerch** [reply]
Excellent work! Thanks for narrowing that down! That's an old XP/Vista view mode feature that I think is enabled via an fflags registry entry. There's a discussion about it [here](https://superuser.com/questions/119921/how-to-display-windows-explorer-column-headers-in-list-mode). I even commented there that the sort options are available via right-click.

At one time, I may have looked at adding that feature as an option in WinSetView, but I probably couldn't get it to work correctly. I can't remember now. I'll have to look at it again and see why it interferes and if there's any way around that.

**Reply by LesFerch** [reply]
I installed Open Shell and enabled "Show sort headers in all views" but everything was still fine. Every folder displayed with the correct view settings for its folder type. Open Shell appears to enable the headers directly in memory. The only registry entries created with that setting are just the settings for Open Shell. No Explorer registry keys were changed.

Have you tried Open Shell and "Show sort headers in all views" with your test user account?

**Reply by Conan179** [reply]
Hatte ich bis eben noch nicht gemacht, habs aber nach geholt, beim test Konto ist der Fehler, auch da

**Reply by LesFerch** [reply]
Then I should be able to replicate the issue. I wonder what I'm missing.

**Reply by Conan179** [reply]
Ich habe es jetzt an einem anderen PC Probiert, Win11 23h2 x64, winsetview innstalliert, nur die einstellungen für Video geändert, Open Shell installiert, getestet, fehler ist nicht da, "Show sort headers in all views" angeschaltet, getestet Fehler da.

**Reply by LesFerch** [reply]
This is frustrating. I still can't replicate it. I'm thinking I need to remote into a test machine of yours to see it first-hand or maybe have you remote into a test machine of mine and see if you can replicate it here.

**Reply by Conan179** [reply]
https://github.com/Open-Shell/Open-Shell-Menu/issues/2023

**Reply by LesFerch** [reply]
I tried again, but I still can't replicate the problem.

**Comment by LesFerch** [comment]
Closing this as caused by the Open Shell setting "Show sort headers in all views" as reported by user, although I couldn't replicate the issue.
